lib/node/rpc.js

```diff
@@ -157,6 +157,8 @@ class RPC extends RPCBase {
     this.add('getblockbyheight', this.getBlockByHeight);
     this.add('getblockhash', this.getBlockHash);
     this.add('getblockheader', this.getBlockHeader);
+    this.add('getblockfilter', this.getBlockFilter);
```

Comment:
This used to be called `getcfilter` in the last iteration of this PR, [I think that is the RPC call for `btcd`](https://github.com/btcsuite/btcd/blob/master/rpcserver.go#L146)

There also was a `getcfilterheader` RPC which `btcd` has, do we need that?

Reply:
I see now, the two data are combined into one response:

```
{
  filter: '014756c0',
  header: '485e301e4509d7f0d954bf5b529f3ecef68c5191fd0e635f775c1d0266dc5a2b'
}
```

I think that's cool! We don't need to copy btcd, do we?

Reply:
Yeah since this BIP wasn't yet implemented in core we didn't have a reference. However from now I think it'd be better to follow core:

https://github.com/bitcoin/bitcoin/blob/08bb4c31563505a30a9b7dd8e7ed98966d2c8494/src/rpc/blockchain.cpp#L2175